.editorconfig

```diff
@@ -8,7 +8,7 @@ root = true
 
 # Change these settings to your own preference
 indent_style = tab
-indent_size = 4
+indent_size = 2
```

Comment:
Doesn't this change the tab_width also making the tab char smaller?

Reply:
reverted